TaskManager.java

```diff
@@ -0,0 +1,71 @@
+@Component
+public class TaskManager {
+    @PersistenceContext
+    private EntityManager entityManager;
+    private static TaskManager instance;
```

Comment:
initiliziation of Taskmanager is not correct, if it is syngleton init there and use double checking

Reply:
Maybe instead of syngleton it is better use Spring bean, with Spring not better idea init yourself